### PR TITLE
Tutorial: How to create a minimap in Duality

### PR DESCRIPTION
**Pull request by ChristianGreiner**
Added a tutorial about creating minimaps in duality.

### CODE REVIEW

File: _posts/2017-09-18-minimap.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+## What we are going to do
+
+One way to implement a minimap is to use a second camera that is positioned above the player and looks towards him.
+This second camera draws everything in the viewport to a `RenderTexture`.
+Later on we are going to render that texture on the `ScreenOverlay`.
+
+## 1. Setup the project
+
+Download Duality from [here](duality.adamslair.net) and install it on your machine. Run `DualityEditor.exe` so it can
+grab all necessary packages from nuget. In this tutorial I'm using the [Kenny Space Shooter Assetpack](http://www.kenney.nl/assets/space-shooter-redux); it's completely free.
```
Comment (mfep) [inline _posts/2017-09-18-minimap.md]: I think we should spell it as 'NuGet'

File: _posts/2017-09-18-minimap.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+The scene view hierarchy should look like this:
+
+![Scene View](https://i.imgur.com/NYcG4Kl.png)
+
+... and the result:
+
+![](https://thumbs.gfycat.com/SoulfulRipeAdamsstaghornedbeetle-size_restricted.gif)
+
+I know, the controls sucks, but it should be fine for this tutorial.
```
Comment (mfep) [inline _posts/2017-09-18-minimap.md]: This line can be omitted imo. The controls are fine for the purpose of this tutorial.

**Review by mfep — COMMENTED** [review]
Thanks for the PR! Suggested minor changes here&there, but other than that, seems solid.

**Comment by ChristianGreiner** [comment]
@mfep Done! :)

**Comment by mfep** [comment]
Thanks! Merging...

**Pull request merged**